wesnoth-ei.po

```diff
@@ -323,22 +325,22 @@ msgstr ""
 #. [achievement_group]
 #: data/campaigns/Eastern_Invasion/achievements.cfg:71
 msgid "Kill no enemies before defeating Mal-Ravanal in ‘<i>The Duel</i>’."
-msgstr ""
+msgstr "Nezabij žádného nepřítele, než v <i>Duelu</i>porazíš Mal-Ravanala."
 
 #. [achievement_group]
 #: data/campaigns/Eastern_Invasion/achievements.cfg:71
 msgid "Scenario 17a: Pacifist"
-msgstr ""
+msgstr "Scénář 17a: Pacifista"
 
 #. [achievement_group]
 #: data/campaigns/Eastern_Invasion/achievements.cfg:73
 msgid "Kill all enemies before defeating Mal-Ravanal in ‘<i>All-In</i>’."
-msgstr ""
+msgstr "Zabij všechny nepřátele, než v <i>All in</i>porazíš Mal-Ravanala."
```

Comment:
Je potřeba přeložit stejně i název scénáře.
```suggestion
msgstr "Zabij všechny nepřátele, než v „<i>Se vším všudy</i>“ porazíš Mal-Ravanala."
```

Reply:
Jasně, ale já jsem se ještě nerozhodl, jaký překlad názvu scénáře zvolím. Proto jsem to také nechal fuzzy.